tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist=
-    py{27,34,35}-{core,coincurve,state-all,state-fast,state-slow,blockchain-fast,blockchain-slow,transactions,vm-all,vm-fast,vm-limits,vm-performance}
+    py{27,34,35}-{core,coincurve, leveldb, state-all,state-fast,state-slow,blockchain-fast,blockchain-slow,transactions,vm-all,vm-fast,vm-limits,vm-performance}
```

Comment:
can you remove the whitespace around `leveldb` to get this inline with the format of the rest of the list.